drivers/can/Kconfig

```diff
@@ -0,0 +1,75 @@
+# Kconfig - CAN configuration options
+
+#
+# Copyright (c) 2018 Alexander Wachter
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+#
+# CAN options
+#
+menuconfig CAN
+	bool
+	prompt "CAN Drivers"
```

Comment:
you can coalesce the type line with the prompt message, it avoids having a dedicated prompt line, it's shorter:
bool "CAN Drivers"

Apply that everwhere in the Kconfig file (you are actually doing it already in some options)

Reply:
Sry. Copy and paste from another driver.